Resources/Prototypes/_DV/Entities/Markers/Spawners/ghost_roles.yml

```
@@ -214,6 +214,9 @@
     description: ghost-role-information-ifritDV-description
     rules: ghost-role-information-ifritDV-rules
     requirements:
+    - !type:DepartmentTimeRequirement
+      department: Epistemics
+      time: 90000 # 25 hours
```

Direction approved except for one thing - 25 hours seems too high. Most pets - such as Laika or Silvia - only require 4 hours, so we'll just go with that.

```suggestion
      time: 14400 # 4 hours
```

---

i understand why and i won't push back on this change (direction knows best) but i will say i disagree that ifrit is similar to laika or silvia because it can ash people and is generally a lot more destructively capable

---

The biggest thing these playtime requirements do is combat raiders from taking the role and doing just that. I get where you're coming from but like I'd probably argue that Epi playtime doesn't really translate to having to know the mechanics of being on fire (medical or atmos playtime would be way better for that).